recipe/meta.yaml

```diff
@@ -29,6 +29,10 @@ requirements:
     - python-confluent-kafka  # [not win]
     - redis-py
     - slack-sdk
+    - globus-sdk
+    - appdirs
+    - pyzbar
```

Comment:
This is already included in the [`analysis`]( https://github.com/nsls-ii-forge/analysis-feedstock/blob/de531d3075e6d016db2c11c06513293c0078e25a/recipe/meta.yaml#L84) metapackage. I suggest removing it from here.